sdk/all/src/test/java/io/opentelemetry/sdk/resources/EnvironmentResourceFuzzTest.java

```diff
@@ -51,6 +52,6 @@ void getAttributesWithFuzzing() {
             "getAttributesWithRandomValues",
             new NoGuidance(10000, System.out),
             System.out);
-    assertThat(result.wasSuccessful()).isTrue();
+    Assertions.assertThat(result.wasSuccessful()).isTrue();
```

Comment:
Shouldn't be needed?
```suggestion
    assertThat(result.wasSuccessful()).isTrue();
```